tests/daily_scraper/test_daily_scraper.py

```diff
@@ -19,46 +19,46 @@
     ({'platform': 'win32', 'branch': 'mozilla-central'},
```

Comment:
Would you mind moving this test to line 32 so that it appears after all the default tests for supported platforms? That way we have a better structure of the tests. The same applies to the other tests as well.